modules/integration_aws-direct-connect/conf/00-heartbeat.yaml

```diff
@@ -0,0 +1,13 @@
+## Example
+module: aws-direct-connect
+name: heartbeat
+
+transformation: false
+aggregation: true
+exclude_not_running_vm: true
```

Comment:
Not needed, this is a cloud service, not running inside a VM

Reply:
Thanks for the feedback, it corrected now. Can you check it plz ?